keps/sig-cloud-provider/providers/0017-cloud-provider-digitalocean.md

```diff
@@ -0,0 +1,75 @@
+---
+kep-number: 17
+title: Cloud Provider DigitalOcean
+authors:
+  - "@andrewsykim"
+owning-sig: sig-cloud-provider
+reviewers:
+  - "@hogepodge"
+  - "@jagosan"
+approvers:
+  - "@andrewsykim"
+  - "@hogepodge"
+  - "@jagosan"
+editor: TBD
+creation-date: 2018-07-23
+last-updated: 2018-07-23
+status: provisional
+
+---
+
+# Cloud Provider DigitalOcean
+
+## Table of Contents
+
+* [Table of Contents](#table-of-contents)
+* [Summary](#summary)
+* [Motivation](#motivation)
+    * [Goals](#goals)
+    * [Non-Goals](#non-goals)
+* [Requirements](#requirements)
+* [Proposal](#proposal)
+
+## Summary
+
+DigitalOcean is a cloud computing platform built for developers and businesses who want a simple way to deploy & manage their infrastructure. This is a KEP proposing DigitalOcean as a cloud provider within the Kubernetes ecosystem.
+
+## Motivation
+
+### Goals
+
+* Supporting DigitalOcean within the Kubernetes ecosystem. This involves:
+  * providing an open community to promote discussions for Kubernetes on DigitalOcean
+  * providing an open environment to development Kuberentes on DigitalOcean
```

Comment:
`Kuberentes` -> `Kubernetes` :)

Reply:
ty :) surprised the spell checker didn't catch that :P